ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1338,6 +1338,16 @@ FCU_DFSS_CHWDC_CSP_RTM_DXZC:
   - filter_alarm
   - run_mode
 
+FCU_SS_RTC_STM_ZTM:
```

Comment:
It is unclear how this equipment controls the return temperature -- it needs another type like DXRC, CHWRC, HTRC, HWRC, etc. (other examples in the ABSTRACT) to describe what component is used to control temperature.

Reply:
Thank you for clarifying - upon further discussion with my team, we believe this is actually a `UH` type instead of a `FCU`. Can you confirm that the unit only runs if it is heating? (For example, the unit would never just run the fan, it will always run the fan + heater together)

If it always runs the fan + heater together, please reclassify as `UH_SS_HTRC_DTM_ZTM`
This would be a unit heater with start/stop, return temp control, discharge temp monitoring (changing from supply since the unit is providing air directly to the space) and zone temp monitoring.

Reply:
Hello, 
This equipment can totally run the fan without heating if the temperature setpoint is reached.
I will try FCU_SS_HTRC_DTM_ZTM